x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -1000,6 +1000,7 @@ export interface PolicyConfig {
       };
     };
     antivirus_registration: {
+      mode?: AntivirusRegistrationModes;
```

Comment:
it is optional only for one release cycle, afterwards it will be required - this is for supporting serverless rollout, see this comment: https://github.com/elastic/kibana/pull/179176#discussion_r1535694587